packages/core/src/components/text-field/text-field.tsx

```diff
@@ -94,8 +97,8 @@ export class TdsTextField {
   tdsInput: EventEmitter<InputEvent>;
 
   // Data input event in value prop
-  handleInput(event): void {
-    this.value = event.target.value;
+  handleInput(event: InputEvent): void {
+    this.value = (event.target as HTMLInputElement).value;
```

Comment:
```suggestion
    handleInput(event: InputEvent): void {
    this.value = (event.target as HTMLInputElement).value;
        this.tdsInput.emit(event);
  }
```

Perhaps we want to avoid "as" and instead do:

```
  handleInput(event: InputEvent): void {
    const { target } = event;
    if (target instanceof HTMLInputElement) {
      this.value = target.value;
      this.tdsInput.emit(event);
    }
  }
```